js/pages/cohort-definitions/cohort-definition-manager.js

```diff
@@ -619,6 +619,10 @@ define(['jquery', 'knockout', 'text!./cohort-definition-manager.html',
 
 				// reset view after save
 					cohortDefinitionService.deleteCohortDefinition(this.model.currentCohortDefinition().id()).then( (result) => {
+						let subscriptions = this.model.cohortDefinitionSubscriptions;
```

Comment:
Question: should these disposals be put into a 'dispose()' method of the view model?  It will be called when the component is removed from the view, which should happen as a result of the 'deleteCohort' action.